erpnext/crm/doctype/linkedin_settings/linkedin_settings.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('LinkedIn Settings', {
+	onload: function(frm){
+		if(frm.doc.session_status == 'Expired' && frm.doc.consumer_key && frm.doc.consumer_secret){
+			frappe.confirm(
+				'Session not valid, Do you want to login?',
```

Comment:
String should be translated, use __("Session not valid, Do you want to login?")

Reply:
Done the changes!